NOTICE

```diff
@@ -0,0 +1,11 @@
+Apache Pekko Persistence Dynamodb
+Copyright 2022, 2023 The Apache Software Foundation
+
+This product includes software developed at
+The Apache Software Foundation (https://www.apache.org/).
+
+This product contains significant parts that were originally based on software from Lightbend (Akka <https://akka.io/>).
+Copyright (C) 2009-2014 Typesafe Inc. [http://www.typesafe.com]
+
+Apache Pekko is derived from Akka 2.6.x, the last version that was distributed under the
```

Comment:
Apache Pekko Persistence DynamoDB is derived from Akka Persistence DynamoDB 1.1.x

Reply:
updated - also remove bit about Business License because the equivalent akka module still is apache licensed - https://github.com/akka/akka-persistence-dynamodb/blob/master/LICENSE